packages/google/google_client.js

```diff
@@ -44,8 +43,13 @@ Google.requestCredential = function (options, credentialRequestCompleteCallback)
         '&scope=' + flatScope +
         '&redirect_uri=' + OAuth._redirectUri('google', config) +
         '&state=' + OAuth._stateParam(loginStyle, credentialToken) +
-        '&access_type=' + accessType +
-        '&approval_prompt=' + approvalPrompt;
+        '&access_type=' + accessType;
+
+  if (typeof options.prompt === 'string') {
+    loginUrl += '&prompt=' + options.prompt;
+  } else if (options.forceApprovalPrompt) {
+    loginUrl += '&prompt=consent';
```

Comment:
Doesn't this actually change the behavior when `forceApprovalPrompt` is passed? Before, it used to send `force`, now it sends `consent`?